flagsmith/resource_feature_state_test.go

```diff
@@ -27,7 +27,17 @@ func TestAccEnvironmentFeatureStateResource(t *testing.T) {
 				Config:      testAccEnvironmentFeatureStateResourceConfig(" some_value ", true),
 				ExpectError: regexp.MustCompile(`Attribute feature_state_value.string_value Leading and trailing whitespace is\n.*not allowed`),
 			},
-
+			// Ensure that empty strings pass validation
```

Comment:
I think we can test with a much simpler test, i.e: something like: 


```
{
	Config:      testAccEnvironmentFeatureStateResourceConfig("", true),
	ExpectError: nil,
			
},
```

Reply:
I updated this as requested